symphony/models/utils.py

```diff
@@ -421,14 +422,23 @@ def model_fn(
             sampling_num_steps=angular_predictor_config.sampling_num_steps,
             sampling_init_step_size=angular_predictor_config.sampling_init_step_size,
         )
-        radial_predictor_fn = lambda: RationalQuadraticSplineRadialPredictor(
-            num_bins=radial_predictor_config.num_bins,
-            min_radius=radial_predictor_config.min_radius,
-            max_radius=radial_predictor_config.max_radius,
-            num_layers=radial_predictor_config.num_layers,
-            num_param_mlp_layers=radial_predictor_config.num_param_mlp_layers,
-            boundary_error=radial_predictor_config.boundary_error,
-        )
+        if config.continuous:
```

Comment:
Copying from elsewhere:
config.continuous is too vague, add this instead to the config.target_position_predictor,
maybe as config.target_position_predictor.continuous_radius